packages/valaxy-theme-press/types/index.d.ts

```diff
@@ -103,9 +103,16 @@ export interface NavItemLink {
   active?: string
 }
 
+export interface NavItemChildren {
```

Comment:
How about we align with the type of vitepress as much as possible?

https://github.com/vuejs/vitepress/commit/e2d4edf45b5ec890c088d3b0517b21a7b3eab9df